ext/js/language/dictionary-importer.js

```diff
@@ -761,7 +761,7 @@ export class DictionaryImporter {
      */
     _normalizeTermOrReading(text) {
         // Note: this function should not perform String.normalize on the text,
-        // as it will characters in an undesirable way.
+        // as it will normalize characters in an undesirable way.
```

Comment:
[sometimes I wonder if I'm even literate in english](https://github.com/themoeway/yomitan/commit/25fe3ba5149cda36e173ee7427324fa74e6784be)